### PR TITLE
added errbit

### LINKED ISSUE #13: Add to errbit
So we can see when things go wrong https://www.ksdetasn.org/blazer/queries/12-is-anyone-using-reported-events

### PR DESCRIPTION
**Pull request by ianfletcher314**
closes #13. Added errbit to this app and got a successful test error:
https://errbit.brandnewops.com/apps/64b1a077ed5fc0000108f0ce/problems/64b1a812ed5fc0000108f0d6

I am updating the docs to reflect that to test errbit you need to run `Airbrake.notify("Test")` from a console rather than `bundle exec rake airbrake:test`

### CODE REVIEW

File: config/initializers/airbrake.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# Airbrake is an online tool that provides robust exception tracking in your
+# Rails applications. In doing so, it allows you to easily review errors, tie an
+# error to an individual piece of code, and trace the cause back to recent
+# changes. Airbrake enables for easy categorization, searching, and
+# prioritization of exceptions so that when errors occur, your team can quickly
+# determine the root cause.
+#
+# Configuration details:
+# https://github.com/airbrake/airbrake-ruby#configuration
+if (project_id = ENV['AIRBRAKE_PROJECT_ID']) &&
```
Comment (ashleyHutton) [inline config/initializers/airbrake.rb]: We don't usually put an `airbrake.rb` config in our project. Was there something in here that you had to update from the default?

Comment (ianfletcher314) [reply config/initializers/airbrake.rb]: After I created the blazer errbit in the directions it said to run `dip rails g airbrake `and i think it came from that command. Should I remove this file?

Comment (ashleyHutton) [reply config/initializers/airbrake.rb]: Yes, go ahead and remove it. It has some duplicate settings from our `errbit.rb`

**Review by ashleyHutton — APPROVED** [review]

**Comment by willtcarey** [comment]
Hey guys, since this is installed as a gem into each of our projects, we don't really want any errbit configuration inside of this gem. Instead you would need to configure the project that it is installed into to be reporting to errbit

**Pull request merged**